vstruct/primitives.py

```diff
@@ -724,3 +724,27 @@ def __repr__(self):
         base = "{%.8x-%.4x-%.4x-%.2x%.2x-%.2x%.2x%.2x%.2x%.2x%.2x}"
         return base  % self._guid_fields
 
+class enum_mixin(object):
```

Comment:
vsGetEnum() ( which returns None or an enum class ) sounds good to me ( on v_number ).